src/screens/TopicPageScreen/index.js

```diff
@@ -68,11 +69,16 @@ const TopicPageScreen = (props) => {
     ? feedsContext.topicFeeds.filter((feed) => feed?.topics?.includes(topicName))
     : [];
   const mainFeeds = feedsContext.feeds;
+  const {timer, viewPostTimeIndex} = feedsContext;
+
   const [offset, setOffset] = React.useState(0);
   const [client] = React.useContext(Context).client;
   const refBlockComponent = React.useRef();
   const scrollY = React.useRef(new Animated.Value(0)).current;
   const {mappingColorFeed} = useCoreFeed();
+
+  const {onWillSendViewPostTime} = useViewPostTimeHook(dispatch, timer, viewPostTimeIndex);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `timer` and `viewPostTimeIndex` are extracted from `feedsContext` but it's not clear where `feedsContext` is defined or imported. Please ensure that `feedsContext` is properly defined and accessible in this scope.
```diff
-  const {timer, viewPostTimeIndex} = feedsContext;
+  const {timer, viewPostTimeIndex} = feedsContext; // Ensure feedsContext is defined
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->